tests/examples/opinions/united_states/colo_example.compare.json

```diff
@@ -7,6 +7,7 @@
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
     "docket_numbers": "21SA286",
+    "citations": "",
```

Comment:
we should probably reference that citation is extracted in extract_from_text.

Reply:
Check out the Kentucky scraper.

If testing - it sets the name to some value.

Reply:
Add default text explaing were citation is going to be extracted from when running tests.

Reply:
We should also update the Colorado court of appeals examples to match the KY example.

Reply:
Added default text for citations form the Appellate court when testing.